app/views/tournaments/new.html.slim

```diff
@@ -1,3 +1,8 @@
+/ #tournament_creation_anchor.col-12
```

Comment:
This looks commented out.  Is that how you are not enabling this right now?  if so, maybe an explanatory comment is in order.